scripts/snowflake/unload_billing_data.sql

```diff
@@ -0,0 +1,180 @@
+-- Existing role with permission to create the necessary objects
+SET CLOUDZERO_BILLING_ACCESS_SCRIPT_ROLE = 'ACCOUNTADMIN';
+
+-- Resources that must already exist outside of Snowflake
+SET CLOUDZERO_BILLING_ACCESS_STORAGE_AWS_ROLE_ARN = '<snowflake-s3-access-role>';
+SET CLOUDZERO_BILLING_ACCESS_S3_UNLOAD_LOCATION = '<bucket>/<prefix>';
+
+-- Objects that will be used if they exist, or created if they do not
+SET CLOUDZERO_BILLING_ACCESS_DATABASE = 'SNOWFLAKE_BILLING_DATA_ACCESS';
+SET CLOUDZERO_BILLING_ACCESS_SCHEMA = 'PUBLIC';
+SET CLOUDZERO_BILLING_ACCESS_WAREHOUSE = 'UNLOAD_SNOWFLAKE_BILLING_DATA';
+SET CLOUDZERO_BILLING_ACCESS_STORAGE_INTEGRATION = 'CLOUDZERO_BILLING_DATA_S3_ACCESS';
+
+-- Objects that be (re)created
+SET CLOUDZERO_BILLING_ACCESS_TASK = 'CLOUDZERO_UNLOAD_BILLING_DATA_TASK';
+SET CLOUDZERO_BILLING_ACCESS_ROLE = 'CLOUDZERO_UNLOAD_BILLING_DATA_ROLE';
+
+----------------------------------------------------------------------------------------------------------------
+/*
+ * Create Storage Integration to give Snowflake access to the S3 bucket where we will unload billing data
+ */
+USE ROLE IDENTIFIER($CLOUDZERO_BILLING_ACCESS_SCRIPT_ROLE);
+CREATE STORAGE INTEGRATION IF NOT EXISTS IDENTIFIER($CLOUDZERO_BILLING_ACCESS_STORAGE_INTEGRATION)
+TYPE = EXTERNAL_STAGE
+  STORAGE_PROVIDER = S3
+  ENABLED = TRUE
+  STORAGE_AWS_ROLE_ARN = $CLOUDZERO_BILLING_ACCESS_STORAGE_AWS_ROLE_ARN
+STORAGE_ALLOWED_LOCATIONS = ('*');
+
+DESC STORAGE INTEGRATION IDENTIFIER($CLOUDZERO_BILLING_ACCESS_STORAGE_INTEGRATION);
+-- NOTE: Must update the IAM Role with STORAGE_AWS_IAM_USER_ARN and STORAGE_AWS_EXTERNAL_ID
```

Comment:
Are these comments obsolete? What is the shape of the change being made to this script? It's hard to tell without a diff